src/lib/index.ts

```diff
@@ -0,0 +1,24 @@
+import {AttrsParser} from './attrs';
```

Comment:
Feels like this file and `extension-load-queue.ts` should be put in their own respective directory.
Then you could use `export * from './extension-load-queue';` in this file, which would be much tidier if more utilities are added.